velox/functions/prestosql/tests/StringFunctionsTest.cpp

```diff
@@ -14,8 +14,11 @@
  * limitations under the License.
  */
 #include <gtest/gtest.h>
+#include <velox/vector/BaseVector.h>
```

Comment:
Do we need these includes?

Reply:
those are added by Vscode, we use them directly but they are included from with in other includes, 
happy to remove them. but this was added for example for BaseVector::ensureWriable(...)